src/webui/www/private/css/Window.css

```diff
@@ -6,173 +6,172 @@ Theme: Default
+    top: 0;
+    left: 0;
 }
 
 /*
 
 	We get a little creative here in order to define a gradient in the CSS using a query
 	string appended to a background image.
-	
+
 	"from" is the top color of the gradient. "to" is the bottom color of the gradient.
-	
+
 	Both must be hex values without the leading # sign.
-  
-*/	
+
+*/
 
 .mochaTitlebar {
-	width: 100%;
-	overflow: hidden;
-	background: url(../images/skin/spacer.gif?from=fafafa&to=e5e5e5); 
+    width: 100%;
+    overflow: hidden;
+    background: url(../images/skin/spacer.gif?from=fafafa&to=e5e5e5);
 }
 
 .mochaTitlebar h3 {
-	font-size: 12px;
-	line-height: 15px;	
-	font-weight: bold;
-	margin: 0;
-	padding: 5px 10px 4px 12px;
-	color: #888;
+    font-size: 12px;
+    line-height: 15px;
+    font-weight: bold;
+    margin: 0;
+    padding: 5px 10px 4px 12px;
+    color: #888;
 }
 
 .mocha.isFocused .mochaTitlebar h3 {
-	color: #181818;
+    color: #181818;
 }
 
 .mochaToolbarWrapper {
-	width: 100%; /* For IE */
-	position: relative;
-	height: 29px;
-	background: #f1f1f1;	
-	overflow: hidden;
-	border-top: 1px solid #d9d9d9;
+    width: 100%;  /* For IE */
+    position: relative;
+    height: 29px;
+    background: #f1f1f1;
+    overflow: hidden;
+    border-top: 1px solid #d9d9d9;
 }
 
 div.mochaToolbarWrapper.bottom {
-	border: 0;
-	border-bottom: 1px solid #d9d9d9;
+    border: 0;
+    border-bottom: 1px solid #d9d9d9;
 }
 
 .mochaToolbar {
-	width: 100%; /* For IE */
-	border-top: 1px solid #fff;
+    width: 100%;  /* For IE */
+    border-top: 1px solid #fff;
 }
 
 .mochaContentBorder {
-	border-top: 1px solid #dadada;
-	border-bottom: 1px solid #dadada;
+    border-top: 1px solid #dadada;
+    border-bottom: 1px solid #dadada;
 }
 
-.mochaContentWrapper { /* Has a fixed height and scrollbars if required. */
-	font-size: 12px;
-	overflow: auto;
-	background: #fff;	
+.mochaContentWrapper {  /* Has a fixed height and scrollbars if required. */
+    font-size: 12px;
+    overflow: auto;
+    background: #fff;
 }
-	
+
 .mochaContent {
-	padding: 10px 12px;
+    padding: 10px 12px;
 }
 
 .mocha .handle {
-	position: absolute;
-	background: #0f0;
-	width: 3px;
-	height: 3px;
-	z-index: 2;
-	-ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=0)"; /* IE8 */		
-	filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=0); /* IE6 and 7*/
-	opacity: .0;	
-	-moz-opacity: .0;
-	overflow: hidden;
-	font-size: 1px; /* For IE6 */
+    position: absolute;
+    background: #0f0;
+    width: 3px;
+    height: 3px;
+    z-index: 2;
+    -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=0)";  /* IE8 */
+    filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=0);  /* IE6 and 7*/
+    opacity: .0;
+    -moz-opacity: .0;
+    overflow: hidden;
+    font-size: 1px;  /* For IE6 */
 }
 
-.mocha .corner { /* Corner resize handles */
-	width: 10px;
-	height: 10px;
-	background: #f00;	
+.mocha .corner {  /* Corner resize handles */
```

Comment:
You added another space in each case of inlined comment. Is it intended changes?

Reply:
A habit of mine to use 2 spaces in front of comment.